docroot/lib/CgovPlatform/Tests/CgovSchemaExclusions.php

```diff
@@ -39,6 +39,7 @@ class CgovSchemaExclusions {
     // for the tracked issue.
     'adobe_dtm.settings',
     'views.view.samlauth_map',
+    'system.site',
```

Comment:
This should not be here. If you tests are complaining about things missing from the schema, you need to fix the schema.